src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -373,6 +374,24 @@ namespace AppInstaller::Runtime
                 }
                 create = true;
                 break;
+            case PathName::PortableLinksUserLocation:
+                result = GetKnownFolderPath(FOLDERID_LocalAppData);
+                result /= s_PortableAppUserRoot;
+                result /= s_LinksDirectory;
+                create = true;
+                break;
+            case PathName::PortableLinksMachineLocationX64:

```

Comment:
Is there a separate system `%PATH%` for x64 and x86?  If there is, I don't think the code to update it is correct.  If there is not, then we don't need separate `PathName`s either.

Reply:
Removed arch separated `%PATH%` to only support one which will be located in the programfiles x64 folder.